.github/workflows/e2e-new-subctl.yml

```diff
@@ -13,10 +13,6 @@ jobs:
       - name: Check out the repository
         uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
 
-      - name: make old subctl
```

Comment:
This whole file should be removed.

Reply:
Perhaps we should keep this and remove the other tests from the [Makefile](https://github.com/submariner-io/subctl/blob/3e7bae502dec0347f8516b823f0eb00d9a29da9c/Makefile#L134). I like the output organized in separate expandable sections. Unless there's a specific reason we want them in the _Makefile_?

Reply:
Having it in the Makefile makes it easy to run locally as well as in CI.